### PR TITLE
Add custom headers in request to docker

### PR DESCRIPTION
**Pull request by rramachand21**

### CODE REVIEW

File: src/Docker.DotNet/Endpoints/ImageOperations.cs [hunk 1]
```diff
@@ -88,6 +93,40 @@ public Task CreateImageAsync(ImagesCreateParameters parameters, Stream imageStre
                 cancellationToken,
                 progress);
         }
+        
+        public Task CreateImageAsync(ImagesCreateParameters parameters, Stream imageStream, AuthConfig authConfig, IDictionary<string, string> headers, IProgress<JSONMessage> progress, CancellationToken cancellationToken = default(CancellationToken))
```
Comment (jterry75) [inline src/Docker.DotNet/Endpoints/ImageOperations.cs]: I'm totally fine with this. Can we update the original `CreateImageAsync` to call this with `null` for `headers` so we dont maintain two identical function bodies?

Comment (rramachand21) [reply src/Docker.DotNet/Endpoints/ImageOperations.cs]: Thanks jterry75, I have updated the pull request

**Comment by jterry75** [comment]
@galvesribeiro - PTAL

**Comment by rramachand21** [comment]
@galvesribeiro @jterry75 could you please create a release with this (latest) changes?

**Comment by jterry75** [comment]
@rramachand21 - Unfortunately at this time we cannot. Our recommend action is to build/sign/release the binaries in your own ingestion pipeline.

**Pull request merged**